modules/gcp/win-gfx/win-gfx-startup.ps1.tmpl

```diff
@@ -218,6 +222,39 @@ function PCoIP-Agent-Register {
     "PCoIP Agent Registered Successfully"
 }
 
+function Install-Idle-Shutdown {
+    "################################################################"
+    "Installing Idle Shutdown..."
+    "################################################################"
+
+    $serviceName = "CAMIdleShutdown"
+    $path = "C:\Program Files\Teradici\PCoIP Agent\bin"
+    $is64 = $true

```

Comment:
is this not needed or did you forget to use it somewhere else?

Reply:
serviceName and is64 not needed since we are always running on 64bit agent. Removed.